egs2/TEMPLATE/asr1/asr.sh

```diff
@@ -409,6 +419,15 @@ if [ -z "${inference_tag}" ]; then
 
     if "${use_k2}"; then
       inference_tag+="_use_k2"
+      if "${inference_with_tlg}"; then
+        inference_tag+="_use_k2_tlg"
+        if ${use_fgram_rescoring}; then
+          inference_tag+="_fgram_rescoring"
+        fi
+      fi
+      if ${use_nbest_rescoring}; then
+        inference_tag+="_nbest_rescoring"
```

Comment:
Is it nbest restoring?
I thought that it is lattice rescoring

Reply:
Sorry.
I think you’re preparing this for neural LMs or attention encoder-decoder rescoring, right?
If so, could you add some comments about it?